prog/install_dnsmasq.rb

```diff
@@ -25,7 +25,7 @@ def compile_and_install
   end
 
   def install_build_dependencies
-    sshable.cmd("sudo apt-get -y install make gcc")
+    sshable.cmd("sudo apt-get -y install make gcc git")
```

Comment:
This seems like an unrelated change, what's up here? Maybe a difference in base image preparation? Does it belong in a separate commit?

Reply:
it was needed since we did install minimal ubuntu but now I set them up with the full image, so this shouldn't be necessary.